<launch>

  <node pkg="sl_sensor_codec" type="generate_patterns_node" name="generate_patterns_node" output="screen">
      <param name="encoder_name" type="string" value="PhaseShiftWithTpu"/>
      <param name="projector_yaml_directory" type="string" value="$(find sl_sensor_projector)/projector_yaml/lightcrafter4500.yaml"/>
      <param name="save_folder" type="string" value="/home/ltf/Desktop/patterns/new2/"/>
      <param name="projector_calibration_data_file" type="string" value="$(find sl_sensor_calibration)/calibration_files/cam2_new2.xml"/>
      <param name="number_phases" type="int" value="12"/>
      <param name="average_value" type="double" value="0.5"/>
      <param name="modulation_intensity" type="double" value="0.5"/>
      <param name="direction" type="string" value="both"/>
  </node>

</launch>
